### ISSUE TITLE
[QUESTION] How to modify the default parameter of setPattRatio function? Is it related to the thickness/width of the borders of square markers?

### ISSUE THREAD

**Issue by Angelo8828** [artoolkit/jsartoolkit5#80]
According to [this guide](https://artoolkit.org/documentation/doku.php?id=3_Marker_Training:marker_training#marker_customizations), the arSetPattRatio is the one that can be used to modify the width of a border's marker. Is it possible to change the width of a border of a square marker using that and if possible, then how?

EDIT: I rephrased the question so that it will become much clearer

**Comment by ThorstenBux** [comment]
Have a look here: https://stackoverflow.com/questions/48143993/change-default-border-thickness-of-a-square-marker-in-artoolkit/48144740#48144740

Also look at ARToolKitX from now on as it is better maintained: https://github.com/artoolkitx/jsartoolkit5

**Comment by ThorstenBux** [comment]
https://github.com/artoolkitx/jsartoolkit5/issues/55

**Comment by Angelo8828** [comment]
Will try this @ThorstenBux, TYVM!

**Comment by Angelo8828** [comment]
@ThorstenBux, hello again. Sorry if I have another question to asked. I already tried it and the magic worked by setting a value for `ARController.setPattRatio()` (which I set first at 0.4, and I prepared markers having different border sizes to have a trial and error testing, the marker having the 30% border worked). I read somewhere ([not sure if here](https://artoolkit.org/documentation/doku.php?id=3_Marker_Training:marker_training)) that the values can only be set from 0-0.5 (if the value decreases, the border size increases, so I thought that the value maybe corresponds to the image size inside the marker). 

My real objective is to actually decrease the border and I tried making the value to 0.6 and it worked again. Now, the border is now only at 20% of the marker length (and it works even if the border is at 15%). Will there be any unintended consequences if the value has been set above 0.5 (I cannot find any other documentation for the arSetPattRatio or the AR_PATT_RATIO explaining why it has been set to 0.5 and not a larger value except for the code itself)?
